h/static/scripts/group-forms/utils/api/fetch-group-annotations.ts

```diff
@@ -0,0 +1,32 @@
+import type { ModerationStatus } from '../../components/ModerationStatusSelect';
+import type { APIConfig } from '../../config';
+import type { APIAnnotationData, GroupAnnotationsResponse } from '../api';
+import { callAPI, paginationToParams } from '../api';
+
+export type FetchGroupAnnotationsOptions = {
+  signal: AbortSignal;
+  pageNumber?: number;
+  moderationStatus?: ModerationStatus;
+};
+
+export async function fetchGroupAnnotations(
```

Comment:
This function mostly just passes parameters through to `callAPI`, so I think it would have been OK to inline the logic in `useGroupAnnotations`. If more complexity is added at some point, then the abstraction will have more benefits.

Reply:
I created the abstraction because it still needs a bit more logic to handle pagination and such.

At some point we may be able to centralize that in `callAPI`.